src/GovUk.Education.ExploreEducationStatistics.Content.Model/Publication.cs

```diff
@@ -14,6 +14,9 @@ public class Publication
 
         [Required(AllowEmptyStrings = false)] public string Title { get; set; } = string.Empty;
 
+        [MaxLength(160)]
```

Comment:
I think this is ok but just pointing out all the other places where we've configured a max length on a table field (not many!) have been configured using the fluent API in `ContentDbContext` or `StatisticsDbContext` rather than with data annotations.